crates/shared/src/lib.rs

```diff
@@ -82,7 +82,8 @@ pub struct ImportStatic {
 
 #[derive(Deserialize, Serialize)]
 pub struct ImportType {
-    pub name: String,
+    pub rust_name: String,
```

Comment:
We actually don't need to change the name here I think, the generated JS bindings don't need to know about the rust name, right? (we just need to be sure to fill in the JS name when this struct is generated)

Reply:
Ah good idea, and that way we don't need to wonder if we have tracked down all the potential bits where we use the wrong version!